packages/sdk/src/adapters/multipeer/session.ts

```diff
@@ -170,6 +180,11 @@ export class Session extends EventEmitter {
 			client.setAuthoritative(false);
 		}
 
+		// forward connection quality metrics
+		if (this.conn instanceof EventedConnection) {

```

Comment:
I don't think this is going to work. The connection quality object cannot be shared. Both connections will update the connection quality, making the values meaningless. What are you trying to accomplish? Maybe there's another way.

Reply:
This is forwarding the latency updates from the authoritative peer to the pipe. See the changes to EventedConnection and Pipe.

---
In reply to: [364934306](https://github.com/microsoft/mixed-reality-extension-sdk/pull/475#discussion_r364934306) [](ancestors = 364934306)

Reply:
When the connection quality tracker object shared, both connections will record latency samples, trashing the calculated average.